PowerFGT/Private/Confirm.ps1

```diff
@@ -104,6 +163,47 @@ Function Confirm-FGTFirewallPolicy {
 
 }
 
+Function Confirm-FGTFirewallProxyPolicy {
+
+    Param (
+        [Parameter (Mandatory = $true)]
+        [object]$argument
+    )
+
+    #Check if it looks like an Firewall Policy element
```

Comment:
FirewallProxyPolicy ? (it the same like Firewall Policy ?)

Reply:
It looks the same, but there is a small difference. There is a check for:
$argument | get-member -name **proxy** -Membertype Properties)